presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -782,6 +784,15 @@ public Plan createPlan(Session session, @Language("SQL") String sql, LogicalPlan
         return createPlan(session, sql, getPlanOptimizers(forceSingleNode), stage);
     }
 
+    public Plan createPlan(Session session, @Language("SQL") String sql, LogicalPlanner.Stage stage, boolean forceSingleNode, WarningCollector warningCollector)
+    {
+        Statement statement = unwrapExecuteStatement(sqlParser.createStatement(sql), sqlParser, session);
```

Comment:
`createStatement()` is deprecated, please use the one that takes a `ParsingOptions`.